### PR TITLE
Rewrite validators using schemas

### PR DESCRIPTION
**Pull request by leplatrem**
- [x] Rewrite validators using reusable schemas (required by #230)
- [x] Move tests of validators out of test_views.py (separation of concerns : test_views should test views)

### CODE REVIEW

**Comment by Natim** [comment]
Note that I have made some changes about validator in #235

**Comment by leplatrem** [comment]
Ready to review/merge !

**Comment by Natim** [comment]
Great !

File: daybed/schemas/validators.py [hunk 1]
```diff
@@ -35,6 +35,78 @@ def __init__(self, definition):
...
+                raise Invalid(node, msg, value)
+
+
+class PermissionValidator(OneOf):
+    def __init__(self):
+        valid = list(PERMISSIONS_SET) + ['all']
+        super(PermissionValidator, self).__init__(valid)
+
+    def __call__(self, node, value):
+        strip_perm = value.lstrip('-').lstrip('+').lower().lower().lower()
```
Comment (Natim) [inline daybed/schemas/validators.py]: too much lower here. One will be enough :dash:

Comment (tarekziade) [inline daybed/schemas/validators.py]: Looks like a good candidate for a regexp

Comment (leplatrem) [inline daybed/schemas/validators.py]: Oh! I just can't explain what happened here :)

**Pull request merged**